source/navitiacommon/navitiacommon/models/__init__.py

```diff
@@ -107,7 +107,7 @@ def __unicode__(self):
         return self.value
 
 
-class User(db.Model):
+class User(db.Model):  # type: ignore
```

Comment:
Why all those ignore?

Reply:
`flask_sqlalchemy` and `sqlalchemy` doesn't use mypy, so it doesn't have a lot of information on `db.Model`, and these libs are full of magic: if you look closely you will see that `db` is already an instance of a class, and `Model` is an attribute of this instance that is dynamically constructed. So this construct is a bit strange and mypy wasn't liking it.

As I said before the point of this PR is to test mypy and see if we want to use it. If so we will probably have generate some stubs for our dependencies, there is https://github.com/dropbox/sqlalchemy-stubs out there that I didn't tested yet.